textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcherTester.java

```diff
@@ -0,0 +1,74 @@
+package edu.uci.ics.textdb.dataflow.regexmatch;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.uci.ics.textdb.api.common.IPredicate;
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.api.dataflow.ISourceOperator;
+import edu.uci.ics.textdb.api.storage.IDataReader;
+import edu.uci.ics.textdb.api.storage.IDataStore;
+import edu.uci.ics.textdb.api.storage.IDataWriter;
+import edu.uci.ics.textdb.common.constants.LuceneConstants;
+import edu.uci.ics.textdb.common.constants.TestConstants;
+import edu.uci.ics.textdb.dataflow.common.RegexPredicate;
+import edu.uci.ics.textdb.dataflow.source.ScanBasedSourceOperator;
+import edu.uci.ics.textdb.dataflow.utils.TestUtils;
+import edu.uci.ics.textdb.storage.LuceneDataStore;
+import edu.uci.ics.textdb.storage.reader.LuceneDataReader;
+import edu.uci.ics.textdb.storage.writer.LuceneDataWriter;
+import edu.uci.ics.textdb.api.common.Attribute;
+
+
+/**
+ * @author zuozhi
+ * 	
+ * Helper class to quickly create unit test
+ */
+public class RegexMatcherTester {
+
+	private RegexMatcher regexMatcher;
+	private IDataWriter dataWriter;
+	private IDataReader dataReader;
+	private IDataStore dataStore;
+	
+	private List<ITuple> result;
+	
+	public RegexMatcherTester(List<Attribute> schema, List<ITuple> data) throws Exception {
```

Comment:
the expected results correspond with a specific regex. I feel adding regex, field name, and expected results to the constructor is too much. 

constructor: initialization, write data to lucene
runTest: run test specific to a regex
matchExpectResults: check if results match

It makes sense to put runTest and matchExpectResults in a same function, but runTest returning a boolean seems wired to me